src/smc-webapp/editor_chat.cjsx

```diff
@@ -126,6 +126,7 @@ class ChatActions extends Actions
 
         @syncdb.save()
         @setState(last_sent: mesg)
+        return new Date(time_stamp).valueOf()
```

Comment:
Why does this return the `valueOf` from a new date object and then get converted into a string later?